config/configgrpc/configgrpc.go

```diff
@@ -116,14 +116,22 @@ type GRPCServerSettings struct {
 
 	// Configures the protocol to use TLS.
 	// The default value is nil, which will cause the protocol to not use TLS.
-	TLSCredentials *configtls.TLSServerSetting `mapstructure:"tls_credentials, omitempty"`
+	TLSCredentials *configtls.TLSServerSetting `mapstructure:"tls_credentials,omitempty"`
 
 	// MaxRecvMsgSizeMiB sets the maximum size (in MiB) of messages accepted by the server.
-	MaxRecvMsgSizeMiB uint64 `mapstructure:"max_recv_msg_size_mib,omitempty"`
+	MaxRecvMsgSizeMiB uint64 `mapstructure:"max_recv_msg_size_mib"`
 
 	// MaxConcurrentStreams sets the limit on the number of concurrent streams to each ServerTransport.
 	// It has effect only for streaming RPCs.
-	MaxConcurrentStreams uint32 `mapstructure:"max_concurrent_streams,omitempty"`
+	MaxConcurrentStreams uint32 `mapstructure:"max_concurrent_streams"`
+
+	// The WriteBufferSize for client gRPC. See grpc.ReadBufferSize
+	// (https://godoc.org/google.golang.org/grpc#ReadBufferSize).
+	ReadBufferSize int `mapstructure:"read_buffer_size"`
```

Comment:
Does this make any difference? In my similar experiments with WebSockets reading buffers did not make any difference since we are reading in big chunks. But perhaps gRPC is different, it will be interesting to know if increasing this buffer size makes a difference.

Reply:
I saw some difference in gRPC on my Mac not as much as on the client but still something.